staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -271,13 +271,13 @@ func (s *store) Delete(
 	if err != nil {
 		return fmt.Errorf("unable to convert output object to pointer: %v", err)
 	}
-	return s.conditionalDelete(ctx, preparedKey, out, v, preconditions, validateDeletion, cachedExistingObject)
+	return s.conditionalDelete(ctx, preparedKey, out, v, preconditions, validateDeletion, cachedExistingObject, true)
```

Comment:
passing booleans is always hard to read without remembering what the 9th parameter of a method means. Any way to make this easier?

E.g.
```golang
transformDecode := true
```

or via constants.